lib/reporting/protocols_report.rb

```diff
@@ -210,6 +215,48 @@ def saml_signature_issues_table
       ]
     end
 
+    def loa_acr_requests_table
+      [
+        ['Count of integrations using LOA', 'List of issuers with the issue'],
+        [
+          loa_issuers_data.length,
+          loa_issuers_data.join(', '),
+        ],
+      ]
+    end
+
+    def loa_issuers_data
+      @loa_issuers_data ||= begin
+        cloudwatch_client.fetch(
+          query: loa_issuers_query,
+          from: time_range.begin,
+          to: time_range.end,
+        ).
+          map { |slice| slice['issuer'] }.
```

Comment:
we don't need to fix this in this PR, but i have been having a hard time with the word `slice` in this context (and other places it's used in maps) especially since we have a different `slice` var on the instance. we should probably go back and change the references to something that make more sense in the local context, but it's not blocking.

Reply:
I inherited the naming from `mfa_report.rb`. The fetched results from CloudWatch are _row slices_, ie they could be a full row from the original query (if the time slice and the period coincided) or they could be a row split into time slices. We could use `row` or `row_slice`. I don't like `row` as it hides the fact that it could have been split up. I would prefer `row_slice`, or leaving it as `slice`.

Reply:
i understand why `slice` was used. i still think reusing the variable `slice` can be confusing at a glance.